apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -632,4 +632,95 @@ describe('LIVECHAT - contacts', () => {
 			expect(res.body.errorType).to.be.equal('invalid-params');
 		});
 	});
+
+	describe('[GET] omnichannel/contacts.search', () => {
+		let contactId: string;
+		const contact = {
+			name: faker.person.fullName(),
+			emails: [faker.internet.email().toLowerCase()],
+			phones: [faker.phone.number()],
+			contactManager: agentUser?._id,
+		};
+
+		before(async () => {
+			await updatePermission('view-livechat-contact', ['admin']);
+			const { body } = await request
+				.post(api('omnichannel/contacts'))
+				.set(credentials)
+				.send({ ...contact });
```

Comment:
```suggestion
				.send(contact);
```
contact already an object and youre not adding any extra key to it.